charts/bundler/templates/static_config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}-static-data
+  namespace: {{ .Release.Namespace }}
+data:
+  {{ $currentScope := . }}
+  {{ range $path, $_ := .Files.Glob "static/*.json" }}
+  {{ . }}: |-
+    {{- with $currentScope}}
```

Comment:
Why do you need to set the scope here?

Reply:
because I was blindly copying their example?

Reply:
Oops, not sure how I managed to comment twice on that line :grimacing: